src/components/Input/Input.tsx

```diff
@@ -88,31 +114,56 @@ const Input = ({
     }
   }, [takeFocus]);
 
-  return (
-    <Field
-      caution={caution}
-      className={wrapperClassName}
-      error={error}
-      forId={id}
-      help={help}
-      label={label}
-      labelClassName={labelClassName}
-      labelFirst={labelFirst}
-      required={required}
-      stacked={stacked}
-      success={success}
-    >
-      <input
-        className={classNames("p-form-validation__input", className)}
+  if (type === "checkbox") {
+    return (
+      <CheckboxInput
+        label={label}
+        defaultChecked={defaultChecked}
         id={id}
-        ref={inputRef}
-        required={required}
-        type={type}
-        aria-invalid={!!error}
+        disabled={disabled}
+        indeterminate={indeterminate}
         {...inputProps}
-      />
-    </Field>
-  );
+      ></CheckboxInput>
+    );
+  } else if (type === "radio") {
+    return (
+      <RadioInput
+        label={label}
+        name={name}
+        defaultChecked={defaultChecked}
+        id={id}
+        disabled={disabled}
+        labelFirst={labelFirst}
+        {...inputProps}
+      ></RadioInput>
+    );
+  } else {
+    return (
+      <Field
+        caution={caution}
+        className={wrapperClassName}
+        error={error}
+        forId={id}
+        help={help}
+        label={label}
+        labelClassName={labelClassName}
+        labelFirst={labelFirst}
+        required={required}
+        stacked={stacked}
+        success={success}
+      >
+        <input
```

Comment:
@bethcollins92 Initially I thought that we would put the `CheckboxInput` or `RadioInput` here inside the `Field` - now I'm not sure if that makes sense - does `Field` even support having checkbox/radio in it?

It seems that having special cases for checkbox/radio in this component only makes sense if it is inside the Field. If it's not I'm not sure if we want radio/checkbox in this component (if it will just be rendering CheckboxInput/RadioInput without the Field).